src/components/OcAppSideBar.vue

```diff
@@ -2,14 +2,14 @@
   <div class="oc-app-sidebar">
     <div class="sidebar-container">
       <div class="header primary">
-        <div class="sidebar-title">
-          <slot name="title">
-            <span>Heading</span>
-          </slot>
-        </div>
         <div v-if="!disableAction" class="action">
           <slot name="action">
-            <button @click="$emit('close', $event)"><v-icon color="white" large>close</v-icon></button>
+            <oc-button icon="close" class="uk-float-right uk-margin-small-right" @click="$emit('close', $event)"></oc-button>
+          </slot>
+        </div>
+        <div class="sidebar-title">
```

Comment:
I don't think we need this component at all - should come from ods - right?

Reply:
in a subsequent PR